src/Esquio.AspNetCore/Blazor/Feature.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Esquio.Abstractions;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Rendering;
+using Microsoft.Extensions.Primitives;
+using System.Threading.Tasks;
+
+namespace Esquio.AspNetCore.Blazor
+{
+    public class Feature : ComponentBase
+    {
+        private bool isEnabled;
```

Comment:
:-) I know you don't like it but we use underscore on all code base!

Reply:
Done!